src/api/app/models/decision.rb

```diff
@@ -23,8 +23,8 @@ def description
 
   # List of all viable types for a reportable, used in the decision creation form
   def self.types(reportable)
-    TYPES.filter_map do |type|
-      [type.display_name, type.name] if type.display?(reportable)
+    TYPES.filter_map do |decision_type|
+      [decision_type.display_name, decision_type.name] if decision_type.display?(reportable)
```

Comment:
These changes belong to the commit before "Use a more descriptive way to label types in decision".